runner/runner_test.go

```diff
@@ -0,0 +1,101 @@
+package runner
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/trento-project/trento/runner/mocks"
+)
+
+// TODO: This test could be improved to check the definitve ansible files structure
+// once we have something fixed
+func TestCreateAnsibleFiles(t *testing.T) {
+	tmpDir, _ := ioutil.TempDir(os.TempDir(), "trentotest")
+	err := createAnsibleFiles(tmpDir)
+
+	assert.DirExists(t, path.Join(tmpDir, "ansible"))
+	assert.NoError(t, err)
+
+	os.RemoveAll(tmpDir)
+}
+
+func TestNewAnsibleMetaRunner(t *testing.T) {
+
+	cfg := &Config{
+		AnsibleFolder: "../test/ansible_test",
+		AraServer:     "127.0.0.1",
+	}
+
+	cmdCallback := exec.Command("echo", "callback")
+	cmdAction := exec.Command("echo", "action")
+
+	mockCommand := new(mocks.CustomCommand)
+	customExecCommand = mockCommand.Execute
+	mockCommand.On("Execute", "python3", "-m", "ara.setup.callback_plugins").Return(
+		cmdCallback,
+	)
+	mockCommand.On("Execute", "python3", "-m", "ara.setup.action_plugins").Return(
+		cmdAction,
+	)
+
+	a, err := NewAnsibleMetaRunner(cfg)
+
+	expectedMetaRunner := &AnsibleRunner{
+		Playbook: "../test/ansible_test/ansible/meta.yml",
```

Comment:
It may be better to try to declare constants on top of the file with the Ansible playbooks directories and file names to avoid repeating them several times. No big deal though.

Reply:
Improved in: https://github.com/trento-project/trento/pull/165/commits/0b54d0ca6fabbbec0f7ed5ab6468d492c123871a